toolkit/tools/internal/userutils/userutils_test.go

```diff
@@ -63,3 +66,78 @@ func TestPasswordExpiresDaysIsValidTooLarge(t *testing.T) {
 	assert.ErrorContains(t, err, "invalid")
 	assert.ErrorContains(t, err, "PasswordExpiresDays")
 }
+
+func TestHashPasswordEmpty(t *testing.T) {
+	hashedPassword, err := HashPassword("")
+	assert.NoError(t, err, "hash password")
+	assert.Equal(t, "", hashedPassword)
+}
+
+func TestHashPasswordNotEmpty(t *testing.T) {
+	hashedPassword, err := HashPassword("password")
```

Comment:
Thoughts on adding a check if `openssl` exists or not, and skipping if it doesn't? I'm sure 99% of people do have it though...

Reply:
I'm just going to add it to the list of dependencies.